### PR TITLE
DRAFT Compiler FE: Enable Transpose conv operator

### PR DESCRIPTION
**Pull request by d-krylov**
EnableTranspose conv operator in luci import, tests; res; tflchef

ONE-DCO-1.0-Signed-off-by: Denis Krylov <krylovdenism@gmail.com>

### CODE REVIEW

File: compiler/tflchef/tflite/src/Op/TransposeConv.cpp [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+  const tflite::Tensor *tensor = import->tensors()->Get(inputs[0]);
+  assert(tensor->type() == tflite::TensorType::TensorType_INT32);
+  const tflite::Buffer *buffer = import->buffers()->Get(tensor->buffer());
+  auto vec = extract_buffer<int32_t>(buffer);
+  import->set_tensor_filler(inputs[0], vec);
+
+  tensor = import->tensors()->Get(inputs[1]);
+  assert(tensor->type() == tflite::TensorType::TensorType_FLOAT32);
+  buffer = import->buffers()->Get(tensor->buffer());
```
Comment (binarman) [inline compiler/tflchef/tflite/src/Op/TransposeConv.cpp:37]: could you check that input is constant before you copy data from it?
like in gather operator
```
  const tflite::Buffer *buffer = import->buffers()->Get(tensor->buffer());

  if (buffer && buffer->data())
  {
    auto vec = extract_buffer<int32_t>(buffer);
    import->set_tensor_filler(inputs[1], vec);
  }
```

Comment (binarman) [reply compiler/tflchef/tflite/src/Op/TransposeConv.cpp:37]: I am also not sure we need this filler for input[1]

File: compiler/luci/import/src/Nodes/CircleTransposeConv.cpp [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  return true;
+}
+
+CircleNode *CircleTransposeConvGraphBuilder::build_node(const circle::OperatorT &op,
+                                                        const std::vector<CircleNode *> &inputs,
+                                                        loco::Graph *graph) const
+{
+  auto *node = graph->nodes()->create<CircleTransposeConv>();
+
+  assert(inputs.size() == 3);
```
Comment (seanshpark) [inline compiler/luci/import/src/Nodes/CircleTransposeConv.cpp:42]: This isn't needed here. `validate()` purpose is to check conditions.

**Comment by seanshpark** [comment]
Plesae check my comments and proceed with sending PRs

**Comment by seanshpark** [comment]
@binarman , can we close this?

**Comment by binarman** [comment]
Everything from this PR is merged, closing.
Thank you!

**Pull request closed without merging**